tests/test_datasource.py

```diff
@@ -169,7 +174,7 @@ def test_load_raw_data(self, dataset, feature_col, target_col):
     def test_load_data(self, dataset, feature_col, target_col):
         """"""
         if isinstance(dataset, HuggingFaceDataset):
-            samples = dataset.load_data(split="test[:30]")
+            samples = dataset.load_data(split="test")
         else:
```

Comment:
Any reason why you remove the `[:30]`?

Reply:
I was trying to fix a broken test then just kept it.